app/src/main/java/org/schabi/newpipe/Downloader.java

```diff
@@ -40,10 +40,13 @@
 
 public class Downloader implements org.schabi.newpipe.extractor.Downloader {
     public static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:43.0) Gecko/20100101 Firefox/43.0";
+    private static final String ACCEPT_LANGUAGE = "Accept-Language";
 
     private static Downloader instance;
     private String mCookies;
     private OkHttpClient client;
+    private static String countryCode = "GB";
```

Comment:
Still not stateless. I think the country, and language should be controlled by the back end, not by the downloader, which is part of the frontend.